data-serving/scripts/converters.py

```diff
@@ -115,3 +116,54 @@ def convert_demographics(id: str, age: str, sex: str) -> Dict[str, Any]:
 
     # If the dictionary is empty, we want to omit it from the JSON output
     return demographics if demographics else None
+
+
+def convert_location(id: str, location_id: float, country: str, adminL1: str, adminL2: str, locality: str, latitude: float, longitude: float) -> Dict[str, Any]:
+    '''Converts location fields to a location object.'''
+    location = {}
+
+    try:
+        if pd.notna(location_id):
+            location['id'] = float(location_id)
+    except (TypeError, ValueError):
+        logging.warning(
+            '[%s] [location.id] invalid value %s', id, location_id)
+
+    if pd.notna(country):
+        location['country'] = country
+
+    if pd.notna(adminL1):
+        location['administrativeAreaLevel1'] = adminL1
+
+    if pd.notna(adminL2):
+        location['administrativeAreaLevel2'] = adminL2
+
+    if pd.notna(locality):
+        location['locality'] = locality
+
+    geometry = {}
+
+    try:
```

Comment:
Abstracting some of these conversion subroutines (e.g. `convertIfNotNa`, `convertIfNotNaFloat`) is something to consider as we continue to add conversion logic.

Reply:
fair, I'll think about it with the next PR! thanks!